builder/layouts/new.hbs

```diff
@@ -1,8 +1,27 @@
-{{> header }}
-<style>
-    iframe {
-        width: 100%;
-        height: 100%;
+{{#> head }}
+  <title>Add new snippet</title>
+  <style>
+    .layout {
+      display: flex;
+      flex-direction: column;
     }
-</style>
-<iframe src="https://codelab.fun/30/" frameborder="0"></iframe>
+
+    .layout__form {
+      flex: 1;
+    }
+    .layout__form iframe {
+      width: 100%;
+      height: 100%;
+    }
+  </style>
+{{/head }}
+
+{{#> body class="layout" }}
+  <div>
+    {{> header }}
+  </div>
+
+  <div class="layout__form">
```

Comment:
Lol, maybe we should either have bem everywhere or nowhere?

Reply:
> maybe we should either have bem everywhere or nowhere?

generally, I'm pro BEM, but refactor everything into it right now is too much imo

Reply:
actually i don't care looks like mdl thing is using it